content/blog/2021-03-10_c2b.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+title: "Cases2Beds: A Case Study in Actionable Intelligence"
+author: "Ananya Joshi, Roni Rosenfeld, and Chris Scott"
+date: 2021-03-10
+tags:
+  - hospital utilization
+  - actionable intelligence
+  - Excel
+  - Allegheny County
+authors:
+  - ajoshi
+  - roni
+  - cscott
+heroImage: /blog/images/blog-lg-c2b.jpg # size: 1120x440 jpg and webp format
+heroImageThumb: /blog/images/blog-thumb-c2b.jpg # size: 300x200 jpg and webp format
+# related:
+summary: | 
```

Comment:
note that the summary will be cropped after a number of words/characters:

![image](https://user-images.githubusercontent.com/4129778/110809489-37998000-8285-11eb-9f31-cf92dc6a1b7d.png)